operatorconfig/moduleconfig/replication/v1.6.0/controller.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dell-replication-controller
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dell-replication-controller-sa
+  namespace: dell-replication-controller
+secrets:
+- name: replication-secret
+---
```

Comment:
How come these and all of the similar resource changes are being added?

Reply:
These were the changes that went in after csm-replication 1.3.0
I.6.0 files are more or less same copy of 1.5.0 but git shows difference like this since 1.3.0 files are deleted.